src/Data/Units.purs

```diff
@@ -148,7 +148,7 @@ type BaseUnitWithExponent = { prefix   ∷ Prefix
 -- | A `DerivedUnit` is a product of `BaseUnits`, raised to arbitrary powers.
 -- | Each factor also has a `Prefix` value which represents a numerical
 -- | prefix as a power of ten or two.
-data DerivedUnit = DerivedUnit (List BaseUnitWithExponent)
+newtype DerivedUnit = DerivedUnit (List BaseUnitWithExponent)
```

Comment:
Could you please clarify why this is beneficial?

Reply:
Well, since a derived unit is, by definition (at least in Insect), a list of units with an exponent each, it seemed to me that a `newtype` shows that equivalence perfectly. Also, it comes with an incredibly minor speed and size improvement, which we shouldn't probably care much about.